_includes/manuals/2.0/3.1.1_supported_platforms.md

```diff
@@ -8,16 +8,17 @@ The following operating systems are supported by the installer, have packages an
     * check the above repositories because the command can silently fail when subscription does not provide it: `yum repolist`
   * Apply all SELinux-related errata.
 * CentOS, Scientific Linux or Oracle Linux 7
+  * **Note:** The CentOS packages are not tested on Scientific Linux or Oracle Linux. The Foreman installation on Scientific Linux or Oracle Linux may or may not work.
   * Architectures: x86_64 only
   * [EPEL](http://fedoraproject.org/wiki/EPEL/FAQ#How_can_I_install_the_packages_from_the_EPEL_software_repository.3F) is required
 * Ubuntu 18.04 (Bionic)
   * Architectures: i386, amd64, aarch64
 * Debian 10 (Buster)
   * Architectures: i386, amd64, aarch64
 
-It is assumed to use the PostgreSQL version provided by the OS, which is currently:
+PostgreSQL from Software Collections:
 
-* PostgreSQL 9.2 or newer
+* PostgreSQL version 10
```

Comment:
This should be 12 i believe (even though 10 should work)

Reply:
Debian Stretch ships version 11 and Ubuntu Bionic ships version 10.

Reply:
So should I mention it separately for Centos, Debian and Ubuntu ? Which is supported and tested pgsql 10 / 12 ?

Reply:
No, PostgreSQL is PostgreSQL. We don't make a distinction between the OS it runs on. It just needs to be 10+. This also relates to a remote DB which may be on a different OS.